docs/docs/firmware/state-monitor-esp32.md

```diff
@@ -99,6 +99,22 @@ To configure input 4 to be a contact sensor, and input 7 to be an inverted butto
 A retained message will ensure the device auto-configures on startup.
 :::
 
+### Recommended Configurations
+Below is a table showing possible **Connected Devices** and the supported `input types`. Check marks are indicating the recommended combinations to ensure intended behavior.
+
+|Connected Devices|`button`|`contact`|`press`|`rotary`|`switch`|`toggle`| 
+|:----------------|:------:|:-------:|:-----:|:------:|:------:|:------:|
+|**bi-stable Switches**   |:x:|:x:|:x:|:x:|:white_check_mark:|:white_check_mark:|
+|**Door / Window Contact**|:x:|:white_check_mark:|:x:|:x:|:x:|:x:|
+|**PIR**                  |:x:|:x:|:white_check_mark:|:x:|:x:|:x:|
+|**Push Button**          |:white_check_mark:|:x:|:white_check_mark:|:x:|:x:|:x:|
+|**Rotary Encoder**       |:x:|:x:|:x:|:white_check_mark:|:x:|:x:|
+
+> **Recommended:**  :white_check_mark:

```

Comment:
Not sure we need this legend do we? You have already mentioned the check marks in the comment above the table.

Reply:
removed left over